core/src/main/java/org/infinispan/stats/impl/ClusterCacheStatsImpl.java

```diff
@@ -199,6 +199,9 @@ public long getAverageWriteTimeNanos() {
       return getStatAsLong(AVERAGE_WRITE_TIME_NANOS);
    }
 
+   @ManagedAttribute(description = "Required minimum number of nodes to guarantee data consistency",
```

Comment:
```suggestion
   @ManagedAttribute(description = "Required minimum number of nodes to hold current cache data",
```

I know this is copied from `CacheMgmtInterceptor`/`Stats`, but it's not really true: if the cluster has 5 nodes and this attribute says 2 nodes would be enough to hold all the data, killing 3 nodes at once will absolutely lose data in a distributed cache with numOwners < 4. 

The attribute also returns a valid number when the eviction strategy is `REMOVE`, which will also lose data if you reduce the number of nodes and then insert new entries.